masterlist.yaml

```diff
@@ -4371,6 +4371,32 @@ plugins:
       # version: 0.32
       - crc: 0xCFB11F37
         util: 'FO4Edit v4.0.2j'
+  - name: 'DWUK_LexingtonInteriors.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/19120' ]
+    dirty:
+      - <<: *reqManualFix
+        crc: 0xEFA1570A
+        subs: [ "Remove edit 000E081D and 0006496C from LexingtonInteriors.esp. Modify edit xx00266B's X position to -36328.718750 to solve the issue." ]
+    clean:
+      - crc: 0xFD5333DA
+        util: 'FO4Edit v4.0.3'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Horizon'
+          - '[Unofficial Lexington Interiors Loot Reduction Patch Plus](https://www.nexusmods.com/fallout4/mods/44487)'
+        condition: 'active("LexingtonInteriors.esp.esp") and active("Z_Horizon.esp") and not active("DWUK_LexingtonInteriors_Patch.esp")'
+  - name: 'DWUK_LexingtonInteriors_Patch.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/44487' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x6CF68492
+        util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 3
+        udr: 39
+    clean:
+      - crc: 0x2235329E
+        util: 'FO4Edit v4.0.3'
```

Comment:
It's placed in the middle of files for Boston FPS Fix, so you'll need to move it after them.

Reply:
Hadn't noticed. Place them a bit higher, issue should be resolved.